tiny/src/config.rs

```diff
@@ -15,11 +15,13 @@ pub(crate) struct Server {
     /// Address of the server
     pub(crate) addr: String,
 
+    #[serde(default)]
+    pub(crate) alias: Option<String>,
+
     /// Port of the server
     pub(crate) port: u16,
 
     /// Use tls
-    #[serde(default)]
```

Comment:
Was this fixed in your clippy warning commit?

Reply:
This seems to be done in your commit.